infortrend-openstack/eonstor-ds-cinder-driver
# Sync from develop

### Driver
- Use units to calculate size
- Modify LOG format
- Remove ExecuteCommand exception handle
- Refactor CLI Return Code. Use return_code_handler to handle
- Implement cinder api: update_migrated_volume

### Test
- Use assertEqual instead of assert
- Remove check exception string and assert on the exact text
- Add update_migrated_volume test case

### Todo
- Waiting retype between difference host.
